City-of-Helsinki/drupal-employment-services
# Thf 637 other languages index

### Changes

- added Article indexes to so, ua and ru
- added event_index to handle all event indexing
- old events indexes are still in Drupal. 

### Testing (been tested and approved)

- Test with UI- Branch https://github.com/City-of-Helsinki/employment-services-ui/pull/385
- Translate some events to Somali (so), Russian (ru), and Ukrainian (ua) from the existing event article.[Article](https://drupal-tyollisyyspalvelut-helfi.docker.so/ajankohtaista/uutiset/oletko-vailla-opiskelupaikkaa-ohjaamo-auttaa-eteenpain)
- run `drush search-api:clear; drush search-api:rebuild-tracker; drush search-api:index; drush cr;`
- Translate some articles to Somali (so), Russian (ru), and Ukrainian (ua) from the existing article example -> [event](https://drupal-tyollisyyspalvelut-helfi.docker.so/ajankohtaista/tapahtumat/tyohaastattelu-tyopaja-0)
- Create basic page [Link to Basic Page](https://drupal-tyollisyyspalvelut-helfi.docker.so/ua/node/add/page)
- If everything works as planned, verify that the front page is still working. Try all languages  [front page](http://localhost:3000/)
- Verify that event page is still working. Try all languages [Events](http://localhost:3000/ajankohtaista/tapahtumat)


### Testing the event filter

- `make drush-cim`
- `drush edel node --bundle=event`
- `drush linkedevents:sync`
- `drush search-api:clear; drush search-api:rebuild-tracker; drush search-api:index; drush cr;`
- Add Display name's to Language taxonomy and translations to all languages
- https://drupal-tyollisyyspalvelut-helfi.docker.so/admin/structure/taxonomy/manage/event_languages/overview
- Go to back to  UI- Branch https://github.com/City-of-Helsinki/employment-services-ui/pull/385
